app/questionnaire/questionnaire_schema.py

```diff
@@ -1423,3 +1431,8 @@ def get_calculation_block_ids_for_grand_calculated_summary(
         calculation_block=grand_calculated_summary_block,
         source_type="calculated_summary",
     )
+
+
+def is_list_collector_block_editable(block: Mapping) -> bool | None:
+    # Type will always exist at this point
+    return block["type"] == "ListCollector"  # type: ignore
```

Comment:
I went with `bool()` in the end, I heard before we should avoid casting types using type hints and `:`.

Reply:
Yes, we should. If we know a type will always be x, it can be type ignored, without casting to appease mypy. We should update the type hiniting doc if it is not there.

Reply:
ah my bad, the doc doesn't explicitly say don't cast, but agree with that being a sensible addition, and ignoring this occurance 👍

Reply:
I think this comment got re-opened by someone so I'm guessing thats to remove `bool(...)` and type ignore instead? I've no strong feeling whether its updated or this just resolved